server/Dockerfile

```diff
@@ -17,7 +17,7 @@ ARG KEYCLOAK_DIST=https://downloads.jboss.org/keycloak/$KEYCLOAK_VERSION/keycloa
 
 USER root
 
-RUN yum update -y && yum install -y epel-release git && yum install -y jq openssl which && yum clean all
+RUN microdnf update -y && microdnf install -y hostname java-11-openjdk-headless openssl tar which && microdnf clean all
```

Comment:
@douglaspalmer it seems that when we removed `git` from this installation we got the trade-off of not having `gzip` in this image. So if you run `docker build` you get something like this:
```
Keycloak from [download]: https://downloads.jboss.org/keycloak/6.0.1/keycloak-6.0.1.tar.gz
tar (grandchild): gzip: Cannot exec: No such file or directory
tar (grandchild): Error is not recoverable: exiting now
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0tar: Child died with signal 13
tar: Error is not recoverable: exiting now
  0  222M    0 22812    0     0  14919      0  4:20:21  0:00:01  4:20:20 14919
curl: (23) Failed writing body (1764 != 16384)
```
Adding `gzip` to the list of packages, solved the issue to me.